cdap-data-fabric/src/main/java/co/cask/cdap/data2/metadata/dataset/MetadataDataset.java

```diff
@@ -754,7 +751,7 @@ private SearchResults searchByCustomIndex(SearchRequest request) throws BadReque
       }
       @SuppressWarnings("ConstantConditions")
       // stop key should always be the stop key for the namespace, regardless of what the cursor is
-      byte[] stopKey = Bytes.stopKeyForPrefix(namespaceStartKey);
+        byte[] stopKey = Bytes.stopKeyForPrefix(namespaceStartKey);
```

Comment:
accidental?

Reply:
funny. IntelliJ indented it that way. Fixed